_news/2019-07-08-firewall-rule-anomaly-detection.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Anomaly Detection Experiments on Firewall Rule in Forseti
+author: Joe Cheuk
+---
+
+Among security professionals, one way to identify a breach or spurious entity is to detect 
+anomalies and abnormalities in customer’ usage trend. Recently, we launched the 
```

Comment:
Should change `customer’` to `customer`?

Reply:
updated